src/express/error.c

```diff
@@ -112,13 +115,36 @@ static struct heap_element {
 static int ERROR_with_lines = 0;    /**< number of warnings & errors that have occurred with a line number */
 static char * ERROR_string;
 static char * ERROR_string_base;
+static char * ERROR_string_end;
 
 static bool ERROR_unsafe = false;
 static jmp_buf ERROR_safe_env;
 
 
 #define error_file stderr /**< message buffer file */
 
+static int ERROR_vprintf( const char *format, va_list ap ) {
+    int result = snprintf( ERROR_string, ERROR_string_end - ERROR_string, format, ap );
+    if(result < 0) ERROR_string = ERROR_string_end;
```

Comment:
Please add braces to match our coding style guidelines - 
```c
if {
    /*...*/
} else if {
    /*...*/
} else {
    /*...*/
}
```

This is no security library, but I still don't want to [goto fail](https://www.synopsys.com/blogs/software-security/understanding-apple-goto-fail-vulnerability-2/) ;)